constants.js

```diff
@@ -30,3 +30,7 @@ export const quotes = [
   "protect the people you love. call out bullshit at all costs. appreciate free thought in all its glory.",
   "we only want to believe the 'good' things about the people we admire, when what makes them heroes is how they negotiate their flaws.",
 ];
+
+// Percentages to be multiplied against useWindowDimensions() height and width values for setting left and top attributes of absolute positioned Animate components
+export const leftPositionPercentDec = .06;
+export const topPositionPercentDec = .45;
```

Comment:
Change these to the `CAPITAL_SNAKE_CASE` since they are constants. 
Suggestion: const `LEFT_POSITION_PERCENT`